templates/noKibana/blocks.go

```diff
@@ -19,27 +19,231 @@ var Blocks = Object{
+						"index": "false",
+						"type":  "long",
+					},
+					"rewardsForProtocolSustainability": Object{
+						"index": "false",
+						"type":  "keyword",
+					},
+					"rewardsPerBlock": Object{
+						"index": "false",
+						"type":  "keyword",
+					},
+					"totalNewlyMinted": Object{
+						"index": "false",
+						"type":  "keyword",
+					},
+					"totalSupply": Object{
+						"index": "false",
+						"type":  "keyword",
+					},
+					"totalToDistribute": Object{
+						"index": "false",
+						"type":  "keyword",
+					},
+				},
+			},
+			"epochStartShardsData": Object{
+				"properties": Object{
+					"epoch": Object{
+						"index": "false",
+						"type":  "long",
+					},
+					"firstPendingMetaBlock": Object{
+						"index": "false",
+						"type":  "keyword",
+					},
+					"headerHash": Object{
+						"index": "false",
+						"type":  "keyword",
+					},
+					"lastFinishedMetaBlock": Object{
+						"index": "false",
+						"type":  "keyword",
+					},
+					"nonce": Object{
+						"index": "false",
+						"type":  "long",
+					},
+					"pendingMiniBlockHeaders": Object{
+						"properties": Object{
+							"hash": Object{
+								"index": "false",
+								"type":  "keyword",
+							},
+							"receiverShard": Object{
+								"index": "false",
+								"type":  "long",
+							},
+							"senderShard": Object{
+								"index": "false",
+								"type":  "long",
+							},
+							"timestamp": Object{
+								"index":  "false",
+								"type":   "date",
+								"format": "epoch_second",
+							},
+							"type": Object{
+								"index": "false",
+								"type":  "keyword",
+							},
+						},
+					},
+					"rootHash": Object{
+						"index": "false",
+						"type":  "keyword",
+					},
+					"round": Object{
+						"index": "false",
+						"type":  "long",
+					},
+					"scheduledRootHash": Object{
+						"index": "false",
+						"type":  "keyword",
+					},
+					"shardID": Object{
+						"index": "false",
+						"type":  "long",
+					},
+				},
 			},
 			"gasPenalized": Object{
-				"type": "double",
+				"type": "long",
+			},
+			"gasProvided": Object{
+				"type": "long",
+			},
+			"gasRefunded": Object{
+				"type": "long",
 			},
 			"maxGasLimit": Object{
-				"type": "double",
+				"type": "long",
+			},
+			"miniBlocksDetails": Object{
+				"properties": Object{
+					"firstProcessedTx": Object{
+						"index": "false",
+						"type":  "long",
+					},
+					"lastProcessedTx": Object{
+						"index": "false",
+						"type":  "long",
+					},
+					"mbIndex": Object{
+						"index": "false",
+						"type":  "long",
+					},
+				},
+			},
+			"miniBlocksHashes": Object{
+				"type": "keyword",
+			},
+			"nonce": Object{
+				"type": "long",
+			},
+			"notarizedBlocksHashes": Object{
+				"type": "keyword",
+			},
+			"notarizedTxsCount": Object{
+				"index": "false",
+				"type":  "long",
+			},
+			"prevHash": Object{
+				"type": "keyword",
+			},
+			"proposer": Object{
```

Comment:
Is it ok to be long here?

Reply:
yes, because in this case propers in an index.